conftest.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+from tasks.models import Comment, Task, Priority, Status
+from users.models import User, Team, Role
+
+
+@pytest.fixture
+def valid_teams():
+    team_1 = Team.objects.create(
+        name="Team1", description="Best team ever")
+    team_2 = Team.objects.create(
+        name="Team2", description="Best team ever")
+    team_3 = Team.objects.create(
+        name="Team3", description="Best team ever")
+    return team_1, team_2, team_3
```

Comment:
I think you can do it without repeating the same code like:
```suggestion
    return (Team.objects.create(name=f"Team{index}", description="Best team ever") for index in range(1, 4))
```